mix.exs

```diff
@@ -8,7 +8,7 @@ defmodule Backpex.MixProject do
     [
       app: :backpex,
       version: @version,
-      elixir: "~> 1.12",
+      elixir: "~> 1.15",
```

Comment:
@pehbehbeh @Flo0807 I think we should bump this version, but I'm not sure which version we want to "officially" support.

Reply:
Bumped it to 1.16

Reply:
👍